.htmllintrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+	'rules': {
+		'indent-style': 'tabs',
+		'attr-quote-style': 'double',
+		'spec-char-escape': false,
+		'attr-bans': ['align', 'background', 'bgcolor', 'border', 'frameborder', 'longdesc', 'marginwidth', 'marginheight', 'scrolling', 'syle'/*, 'width'*/],
```

Comment:
I think there is a typo to `syle` but do we really need these explicit bans?

Reply:
I took the default option and commented the attributes triggering errors.
See https://github.com/htmllint/htmllint/wiki/Options#attr-bans

Reply:
I see, then you can remove the 2 last values (`'syle'` which was initially `'style'` and `'width'`)